src/main/kotlin/de/bund/digitalservice/useid/credentials/CredentialController.kt

```diff
@@ -24,17 +24,17 @@ internal const val CREDENTIALS_BASE_PATH = "/api/v1/credentials"
 @RestController
 @RequestMapping(CREDENTIALS_BASE_PATH)
 @ConditionalOnProperty(name = ["features.desktop-solution-enabled"], havingValue = "true")
-@Tag(
-    name = "Credentials",
-    description = "WebAuthn credentials are used to authenticate the a user in the widget.",
-)
 class CredentialController(
     private val credentialService: CredentialService,
     private val eventStreamService: EventStreamService,
 ) {
     @PostMapping(produces = [MediaType.APPLICATION_JSON_VALUE])
     @Operation(summary = "Start registration for WebAuthn credentials.")
     @ApiResponse(responseCode = "201", content = [Content()])
+    @Tag(
+        name = "eID-Client",
+        description = "WebAuthn credentials are created in the app to later authenticate the user in the widget.",
```

Comment:
this unfortunately hasn't the desired effect. Endpoints are grouped by tag. The tag can only have one description. The first description detected is used. In this case you would override the description of the `eID-Client` tag (the whole group of endpoints), not the description of the method (which is in line 32). 
<img width="731" alt="Screenshot 2023-04-27 at 15 19 52" src="https://user-images.githubusercontent.com/5744476/234874390-760e379a-467b-408f-8cec-2aa3d84f5aa6.png">

Reply:
So please remove all descriptions here.

Reply:
I moved the descriptions for those and the other tags into custom annotations. This should make it more clear that those are only used once and where to update the descriptions.
<img width="1466" alt="Bildschirmfoto 2023-04-28 um 13 54 20" src="https://user-images.githubusercontent.com/15986613/235140967-3b6b7995-32e3-4444-9a47-ab6c07e41dc3.png">